aten/src/ATen/native/mkldnn/xpu/detail/Attr.h

```diff
@@ -362,4 +355,48 @@ class Attr {
   dnnl::post_ops dnnl_post_ops_;
 };
 
+static inline void construct_attr_by_post_op(
+    const c10::string_view& binary_post_op,
+    double binary_alpha,
+    double input1_scale,
+    int64_t input1_zero_point,
+    const c10::string_view& unary_post_op,
+    const torch::List<std::optional<at::Scalar>>& unary_post_op_args,
+    const c10::string_view& unary_post_op_algorithm,
+    at::native::onednn::Attr& attr) {
+  if (binary_post_op == "none") {
```

Comment:
This function combines both binary and unary post-op fusion. Pls. split it into two functions.